projects/igniteui-angular/src/lib/drop-down/drop-down-item.base.ts

```diff
@@ -56,14 +57,34 @@ export abstract class IgxDropDownItemBase implements DoCheck {
      *
      * ```typescript
      *  let mySelectedItem = this.dropdown.selectedItem;
-     *  let isMyItemSelected = mySelectedItem.isSelected; // true
+     *  let isMyItemSelected = mySelectedItem.selected; // true
      * ```
      */
     @Input()
+    get selected(): boolean {
+        return this._isSelected;
+    }
+
+    set selected(value: boolean) {
+        if (this.isHeader) {
+            return;
+        }
+        this._isSelected = value;
+    }
+
+    /**
+     * @hidden
+     */
+    @Input()
+    @DeprecateProperty(`IgxDropDownItemBase \`isSelected\` property is depracated.\n` +
+        `Use \`selected\` instead.`)
```

Comment:
You're deprecating `isSelected` and `isFocused` properties and introducing correspondent `selected` and `focused` properties. As far as I can see the getters/setters of those properties have the exact same code. In this case I recommend removing the code from the deprecated properties and get/set the ones instead. By doing that you'll accomplish two things: will ensure that if someone in the future changes the code of the new properties' getter/setter the old ones will continue to work correctly; will avoid duplicate code.